typescript/infra/scripts/deploy.ts

```diff
@@ -122,7 +124,16 @@ async function main() {
     );
     deployer = new LiquidityLayerDeployer(multiProvider);
   } else if (module === Modules.TEST_RECIPIENT) {
-    throw new Error('Test recipient is not supported. Use CLI instead.');
+    const addresses = getAddresses(environment, Modules.CORE);
+
+    for (const chain of Object.keys(addresses)) {
+      config[chain] = {
+        interchainSecurityModule:
+          addresses[chain].interchainSecurityModule ??
+          ethers.constants.AddressZero,
```

Comment:
It's not very clear why this is required, can you leave a comment?

Reply:
done